cypress/integration/functional/balances/send.spec.ts

```diff
@@ -0,0 +1,106 @@
+import BigNumber from 'bignumber.js'
+
+context('wallet/send', () => {
+  let address: string
+
+  before(function () {
+    cy.createEmptyWallet(true)
+    cy.getByTestID('balances_row_0_utxo').click()
+    cy.getByTestID('receive_button').click()
+    cy.getByTestID('address_text').then(($txt: any) => {
+      // store random address
+      address = $txt[0].textContent
```

Comment:
```suggestion
```

need to replicate this test across 3 address types with this fixed address instead of generating them from the wallet

- `bech32`: `bcrt1q8rfsfny80jx78cmk4rsa069e2ckp6rn83u6ut9`
- `p2sh`: `2MxnNb1MYSZvS3c26d4gC7gXsNMkB83UoXB`
- `legacy`: `n1xjm9oekw98Rfb3Mv4ApyhwxC5kMuHnCo`